README.md

```diff
@@ -105,15 +146,24 @@ KEEP_MY_INTERNAL_POSTGRES = False
 
 * Save the credentials in a file. You will use them inside the notebook.
 
-### 5. Create a notebook in IBM Watson Studio
+### 5. Create a notebook in IBM Watson Studio on Cloud Pak for Data
+
+* In [Watson Studio](https://dataplatform.cloud.ibm.com/) or your on-premise Cloud Pak for Data, click `New Project +` under Projects or, at the top of the page click `+ New` and choose the tile for `Data Science` and then `Create Project`.
+
+* Using the project you've created, click on `+ Add to project` and then choose the  `Notebook` tile, OR in the `Assets` tab under `Notebooks` choose `+ New notebook` to create a notebook.
 
-* In [Watson Studio](https://dataplatform.cloud.ibm.com/), click `New Project +` under Projects or, at the top of the page click `+ New` and choose the tile for `Data Science` and then `Create Project`.
-* In [Watson Studio](https://dataplatform.cloud.ibm.com/) using the project you've created, click on `+ Add to project` and then choose the  `Notebook` tile, OR in the `Assets` tab under `Notebooks` choose `+ New notebook` to create a notebook.
 * Select the `From URL` tab. [1]
+
 * Enter a name for the notebook. [2]
+
 * Optionally, enter a description for the notebook. [3]
+
 * Under `Notebook URL` provide the following url: [https://raw.githubusercontent.com/IBM/monitor-wml-model-with-watson-openscale/master/notebooks/OpenScale.ipynb](https://raw.githubusercontent.com/IBM/monitor-wml-model-with-watson-openscale/master/notebooks/OpenScale.ipynb) [4]
-* For `Runtime` select the `Default Spark Python 3.6 ` option. [5]
+
```

Comment:
This is actually number 4 and not 5 in the image. Also the image shows Python 3.6 runtime.

Reply:
thx. Done.